### PR TITLE
Implement say block

### PR DESCRIPTION
**Pull request by dekrain**
## Important

Merge this after LLK/scratch-render#49

### CODE REVIEW

**Comment by CosmicWebServices** [comment]
I think they have to have a design discussion about this...

**Comment by dekrain** [comment]
But of rendering side

**Comment by CosmicWebServices** [comment]
Yeah

File: src/sprites/clone.js
Comment (tmickel) [inline src/sprites/clone.js]: do you mean `this.renderer`? I can't imagine this would work as I don't think such a variable exists?

File: src/sprites/clone.js
Comment (tmickel) [inline src/sprites/clone.js]: Let's leave the `console.log` for `type === 'think'`.

**Review by tmickel — CHANGES_REQUESTED** [review]

**Comment by dekrain** [comment]
Done!

**Comment by tmickel** [comment]
As you said, this can't be merged until there is support in the renderer.

**Comment by dekrain** [comment]
But you can approve changes and label this ;)

**Comment by tmickel** [comment]
Once we work out the implementation for the renderer :) We'll typically label things as "pr - good to merge" when merging the PR will leave the project in a usable state - i.e., there are no further dependencies.

**Comment by tmickel** [comment]
Hi @dekrain,
I'm going to close this for now, can re-open once we land the say bubble in the renderer and finalize that API. Thanks again for taking the first try at it.

**Pull request closed without merging**
